hax/setup.py

```diff
@@ -174,7 +174,9 @@ def is_runtime_comment(line: str) -> bool:
             define_macros=[('M0_INTERNAL', ''), ('M0_EXTERN', 'extern')],
             library_dirs=[get_motr_libs_dir()],
             runtime_library_dirs=[get_motr_libs_dir()],
-            libraries=['motr'],
-            extra_compile_args=[x for x in get_motr_cflags() + ['-fPIC']])
+            libraries=['motr', 'gcov'],
```

Comment:
What will be the result of these `gcov` and `-ftest-coverage`?

Will it instrument the c code during the build? If so, is it a good idea to instrument the code always (even in production)? Maybe it is a better idea to enable C coverage by some external flag or environment variable?

Reply:
Yes, it instrument the c code during the build. Added environment variable.